File: data/c#/pattern_6/main.cs
using System;
class Grammind
{
    static void Main()
    {
        int num;
        int k = 1;
        string input;
        Console.Write("Enter the number of rows and columns for the square: ");
        input = Console.ReadLine();

        while (!int.TryParse(input, out num) || num <= 0)
        {
            Console.Write("Must the number be positive, re-enter the number of rows and columns for the square: ");
            input = Console.ReadLine();
        }

        for (int i = 1; i <= num; i++)
        {
            for (int j = 1; j <= num; j++)
            {
                Console.Write(k + " ");
                k++;
            }
            Console.Write("\n");
        }
    }
}

File: data/python/pattern_6/main.py
def checkInput(message):
    while True:
        try:
            userInput = int(input(message))
        except ValueError:
            message = 'Must the number be positive, re-enter the number of rows and columns for the square: '
            continue
        else:
            if userInput <= 0:
                message = 'Must the number be positive, re-enter the number of rows and columns for the square: '
                continue
            else:
                return userInput
                break

#MAIN PROGRAM STARTS HERE:
num = checkInput('Enter the number of rows and columns for the square: ')
k = 1

for x in range(0, num ):
    for y in range(0, num):
        print ('%d ' % (k), end='')
        k += 1
    print()